libr/core/cconfig.c

```diff
@@ -1929,7 +1996,12 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF("asm.cmtpatch", "false", "Show patch comments in disasm");
 	SETCB("bin.strpurge", "false", &cb_strpurge, "Try to purge false positive strings");
 	SETPREF("bin.libs", "false", "Try to load libraries after loading main binary");
-	SETCB("bin.strfilter", "", &cb_strfilter, "Filter strings (?:help, a:scii, e:mail, p:ath, u:rl, 8:utf8)");
+	n = NODECB("bin.strfilter", "", &cb_strfilter);
+	SETDESC(n, "Filter strings");
+	SETOPTIONS(n, "a = only alphanumeric printable", "8 = only strings with utf8 chars", NULL);
```

Comment:
But the function parameters suggested different things to me than just a list of rows. Haviinh a single call to setoptions with all the options would be better